core/src/avm2/globals/XML.as

```diff
@@ -1,4 +1,16 @@
 package {
     [Ruffle(InstanceAllocator)]
-    public final dynamic class XML {}
+    public final dynamic class XML {
+        public function XML(value:* = undefined) {
+            this.init(value);
+        }
+
+        private native function init(value:*): void;
+
+        AS3 native function name():Object;
+        AS3 native function localName():Object;
+        AS3 native function toXMLString():String;
```

Comment:
let's add prototype overrides in this PR too, I'm assuming this works in AS
(if not easily, let's add a comment so we don't forget)